modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -290,36 +290,42 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
     if (!userSigningMaterial.backupNShare) {
       throw new Error('Invalid user key - missing backupNShare');
     }
+
     const MPC = new Ecdsa();
     const signingKey = MPC.keyCombine(userSigningMaterial.pShare, [
       userSigningMaterial.bitgoNShare,
       userSigningMaterial.backupNShare,
     ]);
 
     const userSignShare = await ECDSAMethods.createUserSignShare(signingKey.xShare, signingKey.yShares[3]);
+    const signerShare = userSigningMaterial.bitgoNShare.u + userSigningMaterial.bitgoNShare.chaincode;
```

Comment:
Why are we sending userSigning material encrypted if its available locally while user signs it? unless I am missing something

Reply:
@johnoliverdriscoll can you explain why hsm needs the user's signing material? I'm not sure why there's a need but the first call to hsm requires that